packages/cms-cli/bin/cli.js

```diff
@@ -58,10 +59,11 @@ const argv = yargs
   .command(fetchCommand)
   .command(filemanagerCommand)
   .command(secretsCommand)
+  .command(schemaCommand)
```

Comment:
Can we hide this from help? Looks like we need to do the same for `logs` as well.

Reply:
It is hidden. Check out line 5 in schema.js

Reply:
Ah, cool. Didn't realize that is how `yargs` handles that.